docs/DeveloperGuide.md

```diff
@@ -806,41 +812,162 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the `CampusConnect.jar` file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the `CampusConnect.jar` file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
I think you should replace the word `optimum` by `optimal` in this case because `optimum` tends to be used before a noun

Reply:
sure! fixed in 9c011ccf